core/common/src/main/java/alluxio/uri/Authority.java

```diff
@@ -23,6 +23,8 @@
  */
 public interface Authority extends Comparable<Authority>, Serializable {
   Logger LOG = LoggerFactory.getLogger(Authority.class);
+  Pattern LOGICAL_MASTER_AUTH = Pattern.compile("^([a-zA-Z_\\-0-9.]+)$");
+  Pattern LOGICAL_ZOOKEEPER_AUTH = Pattern.compile("^zk@([a-zA-Z_\\-0-9.]+)$");
```

Comment:
Pointing out that these regex's `[]` groups end with `.` whereas your regexes above for `NAMESERVICE_PATTERN_STRING`, `ALLUXIO_MASTER_ID_PATTERN_STRING`, `ZOOKEEPER_NODE_ID_PATTERN_STRING` end with `\\.`. From what I understand it should match the character `.` literally either way but just letting you know.

Reply:
fix